fe/fe-core/src/main/cup/sql_parser.cup

```diff
@@ -4743,6 +4743,10 @@ function_call_expr ::=
       RESULT = new FunctionCallExpr(fn_name, params);
     }
   :}
+  | function_name:fn_name LPAREN function_params:left_params RPAREN LPAREN function_params:params RPAREN
```

Comment:
I prefer the function grammar  "windowFunnel(window, mode, timestamp, cond1, cond2, ..., condN)"

Reply:
yeah, discussions are needed here on the grammar. I was considering that we could extends the function by adding modes parameter in ck, so I used ()()gramma like ck.  If we use gramma with two brackets, then we can add modes parameters with backward compatibility.

Reply:
Single bracket cant support `Parametric Aggregate Functions` grammar from ck.
Single bracket cannot represent 2 variable length arguments. But maybe we can use some grammar like `(n,a1,a2...an,m,b1,b2..bm)` to represent it.

Reply:
hologres grammar https://help.aliyun.com/document_detail/216948.html

Reply:
From the point of view of function capabilities, single parentheses can meet the needs.
For example, intersect count is currently implemented in this way.
From the perspective of grammar compatibility, it depends on whether you want to follow the consistent syntax style of Doris or cater to CK.
In fact, much of the grammar of CK is not generic.